sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -1942,6 +1942,10 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 				currentVersion = 271;
 				getDao(Exposure.class).executeRaw("ALTER TABLE cases ADD COLUMN caseIdentificationSource varchar(255);");
 
+			case 272:
+				currentVersion = 272;
+				getDao(Location.class).executeRaw("ALTER TABLE location ADD COLUMN country_id BIGINT;");
```

Comment:
Foreign key is missing (= REFERENCES keyword, check an earlier upgrade for an example)